changes/14116-citrix-false-pos

```diff
@@ -0,0 +1 @@
+- Fixes a false positive vulnerabilty report for Citrix Workspace on Windows.
```

Comment:
Nice to have, I'm guessing the issue also happens on macOS and Linux, while we are here, any chance we can fix this there too?

Reply:
Oh I see; I'll add some checks for that

Reply:
@lucasmrod I haven't been able to repro this on MacOS fwiw. Were you able to repro?

Reply:
Seems it's reported as 23.09 instead of 2309:
![Screenshot 2023-11-15 at 19 08 26](https://github.com/fleetdm/fleet/assets/2073526/d86f43bd-9a7f-4045-945a-29fd8b544d8f)